scope-validation-announcement.md

```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
+With the initial release of the ACS we allowed to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript)
+
+However we have not enforced the VoIP and PSTN scopes. Meaning you could make a call with  a chat only scope in your token.
+
+Starting tomorrow we will start rolling out the update to enforce the scopes. That means if you have a token with chat only scope you will not be able to make a call.
```

Comment:
"...update to enforce the 'voip' scope."

Reply:
added